lstchain/reco/dl1_to_dl2.py

```diff
@@ -79,7 +79,8 @@ def train_energy(train, custom_config=None):
 
     reg = model(**energy_regression_args)
     reg.fit(train[features],
-            train['log_mc_energy'])
+            train['log_mc_energy'],
+            sample_weight=train['weight'])
```

Comment:
sorry, I do not understand how this is done... is this way added as a column to the table? how is it calculated?

Reply:
I updated the description

Reply:
https://github.com/cta-observatory/cta-lstchain/blob/a595cee71afec4a7012e77d0395f84941bc44762/lstchain/reco/utils.py#L867
is what computes the weights

Reply:
shouldn't you return the `events` dataframe in this function?

Reply:
> shouldn't you return the `events` dataframe in this function?

That was my first intention, but the dataframe is modified, ,so it is not needed.